server/treetest_server.js

```diff
@@ -59,7 +59,16 @@ module.exports = {
             data: {
                 showSiblings: true,
                 selectableParents: true,
-                tasks: ['Where is the Apple?','Where is the Bacon?'],
+                tasks: [
+                    {
+                        question: 'Where is the Apples?',
+                        expectedAnswers:['Fruits,Apple']
+                    },
+                    {
+                        question: 'Where is the Bacon?',
+                        expectedAnswers:['Meats,Bacon']
```

Comment:
I feel that it might be easier to compare results if we don't collapse the answer into one string and might avoid some complication with delimiters.
```suggestion
                        expectedAnswers: { ['Meats','Bacon'] }
```

What do you think? Is my suggestion an unnecessary overhead?

Reply:
Do you mean:

```suggestion
                        expectedAnswers: [ ['Meats', 'Bacon'] ]
```

It seems like a good idea to me

Reply:
That's not what I meant, but that's what I should have suggested! Thank you! :)

Reply:
Updated the schema to store each answer as an array